src/modules/CircularGradient/Module.js

```diff
@@ -28,42 +34,40 @@ module.exports = function Circular(options, UI) {
             if(width%2 != 0) {
                 width = width-1;
             }
-            console.log('dchanged');
-            var decrease = 0;
             var increment = width/2;
             console.log(width);
             console.log(height);
-            var decrease = 255;
-            var decrement = 1;
-
-            for (var i = 0 ; i < width/2-1 ; i++) {
-                decrement = 1;
-                decrease = 255;
-                for (var j = 0 ; j < height/2-1 ; j++) {
-                    pixels.set(i, j, 0, 0);
-                    pixels.set(i, j, 1, 0);
-                    pixels.set(i, j, 2, 0);
-                    pixels.set(i, j, 3, decrease);
-                    decrease-=decrement;
-                }
-                decrement+=0.2;
+            if (options.r > 210) {
+                options.r = 210;
+            }
+            else if (options.g > 210) {
+                options.g = 210;
+            }
+            else if (options.b > 210) {
+                options.b = 210;
+            }
+            else if (options.r < 0) {
+                options.r = 0;
+            }
+            else if (options.g < 0) {
+                options.g = 0;
+            }
+            else if (options.b < 0) {
+                options.b = 0;
             }
 
-            decrease = 255;
-            decrement = 1;
-            console.log(decrease, decrement);
-
-            for (var i = pixels.shape[0]/2-1 ; i < pixels.shape[0]-1 ; i++) {
-                decrement = 1;
-                decrease = 255;
-                for (var j = pixels.shape[1]/2 ; j > 0 ; j--) {
-                    pixels.set(i, j, 0, 0);
-                    pixels.set(i, j, 1, 0);
-                    pixels.set(i, j, 2, 0);
-                    pixels.set(i, j, 3, 255-decrease);
-                    decrease-=decrement;
+            for (var i = 0 ; i < width ; i++) {
+                for (var j = 0 ; j < height ; j++) {
+                    var distX = Math.abs(options.centreX - i);
+                    var distY = Math.abs(options.centreY - j);
+                    var distance = Math.sqrt(Math.pow(distX, 2) + Math.pow(distY, 2));
+                    var brightness = 255 * (distance / (2*width));
+                    if (distance-1 > width) {distance = width;}
+                    pixels.set(i, j, 0, options.r*1.4*distance/width);
```

Comment:
Then here, you'd just do distance/width, without using a color, and the same for the next 2 lines.